### ISSUE TITLE
Shouldn't this gem be called turbolinks-rails

### ISSUE THREAD

**Issue by mintuhouse** [turbolinks/turbolinks-rails#1]
Based on [comment](https://github.com/basecamp/turbolinks#installation) on turbolinks v5, I am assuming this is the new gem

**Comment by mintuhouse** [comment]
Just letting you know
I tried to install it via
`gem 'turbolinks', github: 'turbolinks/turbolinks-rails'`
but I got the following error
`cannot load such file -- turbolinks/source/version` 
in 

```
turbolinks-source-5.0.0.beta1/lib/turbolinks/source.rb:1
bundler/gems/turbolinks-rails-d10da5ffd478/lib/turbolinks.rb:3
```

I had to install turbolinks-source also from github for it to work, though the turbolinks-source is released on rubygems
`gem 'turbolinks-source', github: 'turbolinks/turbolinks-source-gem'`

**Comment by sstephenson** [comment]
This repository is named `turbolinks-rails`, but for purposes of continuity, the gem will continue to be called `turbolinks`. (For version 5, we’ve split off the main JavaScript source from the Rails plugin.)

**Comment by sstephenson** [comment]
> cannot load such file -- turbolinks/source/version 

Sorry about that. The turbolinks-source gemspec was missing a file. Fixed and released as 5.0.0.beta1.1:
https://github.com/turbolinks/turbolinks-source-gem/compare/2e39d2d301...5a186104bb
